x-pack/legacy/plugins/siem/server/lib/detection_engine/signals/signal_rule_alert_type.ts

```diff
@@ -49,6 +50,7 @@ export const signalRulesAlertType = ({
         threats: schema.nullable(schema.arrayOf(schema.object({}, { allowUnknowns: true }))),
         to: schema.string(),
         type: schema.string(),
+        updatedAt: schema.nullable(schema.string()),
```

Comment:
I'm wondering if we want these to be `nullable` at all or if we want to enforce these are always here like so:

```ts
updatedAt: schema.string()
createdAt: schema.string()
```

Seems like they should always be there from now on and never nullable.